README.md

```diff
@@ -5,17 +5,30 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
 ## Requirements
 - A supported operating system.
 - MySQL or PostgreSQL database in which to store Airflow metadata.
+<<<<<<< HEAD
+- RabbitMQ
+- [Airflow Parcel](https://github.com/teamclairvoyant/apache-airflow-parcels)
+
+### Currently Supported Versions of Airflow
+- Airflow 1.9.0
+- Airflow 1.10.0
```

Comment:
Be sure to add the specific sub-versions that are supported.

Reply:
We built the CSD's for Airflow 1.9.0 and Airflow 1.10.0. Do you want to include all the subversions like Airflow 1.10.1b1?

Reply:
Update to Airflow 1.10.3

Reply:
Great. Thanks